pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -943,9 +963,9 @@ func (r *ReconcileVirtualMachineImport) afterSuccess(vmName types.NamespacedName
 }
 
 //TODO: use in proper places
-func (r *ReconcileVirtualMachineImport) afterFailure(vmiName types.NamespacedName, p provider.Provider) error {
+func (r *ReconcileVirtualMachineImport) afterFailure(vmiName types.NamespacedName, p provider.Provider, instance *v2vv1alpha1.VirtualMachineImport) error {
```

Comment:
same comment - we can pass only the new are 'instance' and initialize vmiName within the method:

Reply:
OK